### PR TITLE
Adding LiteLLM  to AI guides/docs.

### PR DESCRIPTION
**Pull request by ghost**
## What kind of change does this PR introduce?

Adding [LiteLLM (YC W23)](https://github.com/BerriAI/litellm) to AI guides/docs. 

## What is the new behavior?

We added support for how users can instantly log their LLM API requests and monitor costs per provider using Supabase w/ LiteLLM. Teaching users how to make the most of it. 

## Additional context

Attaching a relevant code snippet: 

```python
from litellm import completion

## set env variables
os.environ["SUPABASE_URL"] = "your-supabase-url" 
os.environ["SUPABASE_KEY"] = "your-supabase-key" 
os.environ["OPENAI_API_KEY"] = "your-openai-key"

# set callbacks
litellm.success_callback=["supabase"]
litellm.failure_callback=["supabase"]

#openai call
response = completion(model="gpt-3.5-turbo", messages=[{"role": "user", "content": "Hi 👋 - i'm openai"}]) 

#bad call
response = completion(model="chatgpt-test", messages=[{"role": "user", "content": "Hi 👋 - i'm a bad call to test error logging"}]) 
```

### CODE REVIEW

**Comment by hieuwu** [comment]
Looks like changes in this PR are too large with unnecessary changes. Could you please double check and only update what we need?

File: package-lock.json
Comment (hieuwu) [inline package-lock.json:1]: Looks like changes in this PR are too large with unnecessary changes. Could you please double check and only update what we need?

Comment (krrishdholakia) [reply package-lock.json:1]: @hieuwu cleaned up the commit - let me know if there's anything else you need changed on my end.

**Comment by krrishdholakia** [comment]
@hieuwu Any updates on this? I updated the PR based on your feedback

cc: @nicetomytyuk

**Comment by hieuwu** [comment]
> @hieuwu Any updates on this? I updated the PR based on your feedback
> 
> cc: @nicetomytyuk

None from me. It looks good

**Review by hieuwu — APPROVED** [review]

**Comment by krrishdholakia** [comment]
Great - what're next steps then? anything i can do to help unblock this PR?

**Comment by kiwicopple** [comment]
(discussed on slack - we'll close this one for now)
